src/main/webapp/resources/js/pages/admin/routes.js

```diff
@@ -6,5 +6,9 @@ export const ADMIN = {
   ADMIN: "admin",
   STATISTICS: "statistics",
   USERS: "users",
-  USERGROUPS: "groups"
+  USERGROUPS: "groups",
+  CLIENTS: "clients",
+  REMOTEAPI: "remote_api",
+  SEQUENCINGRUNS: "sequencingRuns",
```

Comment:
Change `SEQUENCINGRUNS: "sequencingRuns"` to `SEQUENCINGRUNS: "sequencing_runs"`